tutor/templates/build/forum/bin/docker-entrypoint.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/sh -e
 
-export MONGOHQ_URL="mongodb://$MONGODB_AUTH$MONGODB_HOST:$MONGODB_PORT/cs_comments_service"
+if test "$MONGODB_PROTOCOL" = 'mongodb+srv'; then
+  export MONGOHQ_URL="$MONGODB_PROTOCOL://$MONGODB_AUTH$MONGODB_HOST/cs_comments_service$MONGODB_PARAMS"
+  export MONGO_WAIT=""
```

Comment:
The `MONGOHQ_URL` variable needs to be exported, but not `MONGO_WAIT`, which is only used in this entrypoint script.